libs/sdk-ui-kit/src/List/LegacySingleSelectList.tsx

```diff
@@ -0,0 +1,108 @@
+// (C) 2007-2020 GoodData Corporation
+import React, { Component } from "react";
+import pick from "lodash/pick";
+import cx from "classnames";
+import { guidFor } from "@gooddata/goodstrap/lib/core/Guid";
+
+import { LegacyList } from "./LegacyList";
+import { LegacyListItem } from "./LegacyListItem";
+import { LegacySingleSelectListItem } from "./LegacySingleSelectListItem";
+
+/**
+ * @internal
+ */
+export interface ILegacySingleSelectListProps<T> {
+    className?: string;
+    getItemKey?: (item: T) => string;
+    height: number;
+    itemHeight: number;
+    items?: T[];
+    itemsCount?: number;
+    listItemClass?: React.ElementType;
+    onItemMouseOut?: () => void;
+    onItemMouseOver?: () => void;
+    onItemMouseEnter?: (id: string) => void;
+    onItemMouseLeave?: () => void;
+    onRangeChange?: () => void;
+    onScrollStart?: () => void;
+    onSelect?: (item: T) => void;
+    scrollToSelected?: boolean;
+    rowItem?: React.ReactElement;
+    selection?: T;
+    width: number;
+}
+
+/**
+ * @internal
+ * @deprecated This component is deprecated use SingleSelectList instead
+ */
+export class LegacySingleSelectList<T> extends Component<ILegacySingleSelectListProps<T>> {
+    static defaultProps = {
+        className: "",
+        getItemKey: guidFor,
+        items: [] as any[],
+        itemsCount: 0,
+        listItemClass: LegacySingleSelectListItem, // TODO add tests
```

Comment:
Is this TODO actionable? Or are the tests necessary now that it is deprecated?

Reply:
removed as you mentioned it is now Legacy component